src/bptl/camunda/tasks.py

```diff
@@ -29,6 +29,8 @@
             (settings.LONG_POLLING_TIMEOUT_MINUTES * 60) + 1
         ),  # timeout if something goes wrong, in seconds
     },
+    exchange="long-polling",
+    queue="long-polling",
```

Comment:
checking the docs at https://docs.celeryproject.org/en/4.4.0/userguide/tasks.html#general it seems like the `queue` option for the `@app.task` decorator is no longer available, and the routing setting should instead be used.

```suggestion
```

I'm a little bit confused by https://docs.celeryproject.org/en/4.4.0/userguide/routing.html#specifying-task-destination though, where 2. seems to imply that the `queue` task option _does_ still work.

Reply:
Me2 :X